react/DeliveryItems.tsx

```diff
@@ -17,7 +16,7 @@ const DeliveryItems: FC = () => {
   }
 
   return (
-    <OrderSection name="deliveryItems">
```

Comment:
Couldn't this break some stores due to styles applied to this section?

Reply:
If I understood you correctly, I've added them as props in the `blocks.jsonc`.

Reply:
Or do you mean for previously styled pages?

Reply:
We weren't using css handles before, so any markup change can break people that customized their order-placed... Dunno what to do here. I'm going to test it on some IO stores that customized their page.